GeosCore/aerosol_mod.F90

```diff
@@ -624,19 +624,15 @@ SUBROUTINE AEROSOL_CONC( Input_Opt,  State_Chm, State_Diag, &
           ! Lump 1st dust tracer for het chem
           ! Now use dust size distribution scheme to improve PM2.5
           ! surface dust conc over western U.S. (L. Zhang, 6/25/15)
-          SOILDUST(I,J,L,1) = 0.007e+0_fp  * Spc(id_DST1)%Conc(I,J,L) &
-                              / AIRVOL(I,J,L)
-          SOILDUST(I,J,L,2) = 0.0332e+0_fp * Spc(id_DST1)%Conc(I,J,L) &
-                              / AIRVOL(I,J,L)
-          SOILDUST(I,J,L,3) = 0.2487e+0_fp * Spc(id_DST1)%Conc(I,J,L) &
-                              / AIRVOL(I,J,L)
-          SOILDUST(I,J,L,4) = 0.7111e+0_fp * Spc(id_DST1)%Conc(I,J,L) &
-                              / AIRVOL(I,J,L)
+          SOILDUST(I,J,L,1) = Spc(id_DSTbin1)%Conc(I,J,L) / AIRVOL(I,J,L)
```

Comment:
Does removal of the scale factors mean we are no longer using the dust distribution scheme mentioned in the comment? If yes, that comment should be removed.